CONTRIBUTING-students.md

```diff
@@ -132,6 +181,39 @@ For example, you can
 $ pandoc -f markdown -t pdf YYYY/proposals/your-name.md
 ~~~
 
+# Organization Instructions
+
+**Please use the instructions of the project/organization
+you are interested in work with.**
+
+## biocore
```

Comment:
You need to comment this organizations until they tell us they will participate.